.travis.yml

```diff
@@ -56,6 +52,15 @@ matrix:
         - composer require --dev phpunit/phpunit ~9.3 --ignore-platform-reqs
         - cd tests/PHPUnit
         - $PIWIK_ROOT_DIR/tests/travis/travis.sh
+    - php: 8.1
+      env: TEST_SUITE=PluginTests MYSQL_ADAPTER=PDO_MYSQL TEST_AGAINST_CORE=minimum_required_piwik SKIP_COMPOSER_INSTALL=1
```

Comment:
@snake14 This should test against `PIWIK_TEST_TARGET` i.e `4.x` or `5.x` right ?

Reply:
@AltamashShaikh I don't think so. Testing against the current branch (`PIWIK_TEST_TARGET`) is handled on [line 47](https://github.com/matomo-org/plugin-Migration/pull/47/files#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485R47). This line is to test against the minimum Matomo version specified in the `plugin.json` file.